src/Microsoft.Diagnostics.Monitoring.StartupHook/Exceptions/Eventing/ExceptionsEventSource.cs

```diff
@@ -148,7 +155,29 @@ internal sealed class ExceptionsEventSource : EventSource
             SetValue(ref data[NameIdentificationEvents.TokenDescPayloads.OuterToken], OuterToken);
             SetValue(ref data[NameIdentificationEvents.TokenDescPayloads.Name], namePinned);
 
-            WriteEventCore(TokenDescriptionEventId, data);
+            WriteEventCore(ExceptionEvents.EventIds.TokenDescription, data);
+
+            RestartFlushingEventTimer();
+        }
+
+        [Event(ExceptionEvents.EventIds.Flush)]
+        public void Flush()
```

Comment:
Should this be public? We would want to either force all the clients to flush after a batch of events, or do it automatically with a timer, but not both.

Reply:
I'll have to check if the event source manifest generation considers internal/private methods.

Reply:
Changed to private